cpp/ql/test/library-tests/dataflow/dataflow-tests/test.cpp

```diff
@@ -594,5 +594,5 @@ void test_indirect_flow_to_array() {
   int* p = indirect_source();
   int* xs[2];
   xs[0] = p;
-  sink(*xs[0]); // $ MISSING: ir,ast
+  sink(*xs[0]); // $ ir=594:12 MISSING: ast SPURIOUS: ir=595:8
```

Comment:
Is this really spurious, or is this just because we're also trying to track flow from uninitialised variables?

Reply:
This is from commit 9533b98797aa7a3dddd58bb2df58032ec5f0d682

Reply:
Yeah, you're right. This is just because of the flow from the uninitialized variable. It's not really unitialized, though, since it's stack-allocated. So I think this might be something that should be fixed as part of https://github.com/github/codeql-c-team/issues/1409